frontend/src/v5/ui/routes/viewer/tickets/ticketDetails/ticketsDetailsCard.component.tsx

```diff
@@ -128,7 +129,7 @@ export const TicketDetailsCard = () => {
 		setTicketId();
 		TicketsCardActionsDispatchers.setCardView(TicketsCardViews.List);
 	}, []);
-
+	if (!ticket) return null;
```

Comment:
Why is this file is changed?

Reply:
There was a crash when navigating backwards to a model with a ticket open in a different project. The tickets card opens before the ticket is defined